integration-test/docker-compose_template.yml

```diff
@@ -33,6 +33,7 @@ services:
       - INTEGRATIONTEST_SDX_SERVER=http://dev-gateway
       - INTEGRATIONTEST_REDBEAMS_SERVER=http://dev-gateway
       - ALTUS_AUDIT_ENDPOINT=thunderhead-mock
+      - ALTUS_DATALAKEDR_ENDPOINT=thunderhead-mock
```

Comment:
you are pointing int tests to the mock, but there is no mock implementation of this service, is there? this could be misleading

Reply:
Removed these settings.